packages/react-components/react-utilities/src/index.ts

```diff
@@ -75,14 +75,14 @@ export type { FluentTriggerComponent, TriggerProps } from './trigger/index';
 export type { NativeTouchOrMouseEvent, ReactTouchOrMouseEvent, TouchOrMouseEvent } from './events/index';
 export { isTouchEvent, isMouseEvent, getEventClientCoords } from './events/index';
 
+export { useSelection } from './selection/index';
 export type {
-  SelectionMode,
   OnSelectionChangeCallback,
   OnSelectionChangeData,
-  SelectionItemId,
   SelectionHookParams,
+  SelectionItemId,
   SelectionMethods,
+  SelectionMode,
 } from './selection/index';
```

Comment:
Are there any changes except sorting? If no, let's set changes to `none` in `change/@fluentui-react-utilities-18dc963b-51db-49d1-aed0-04054579847f.json`.

Reply:
looks like it, i'll revert changes on this file